soroban-env-host/Cargo.toml

```diff
@@ -27,6 +27,8 @@ tinyvec = { version = "1.6.0", features = ["alloc"] }
 dyn-fmt = "0.3.0"
 log = "0.4.17"
 backtrace = "0.3"
+# NB: this must match the same rand version used by ed25519-dalek above
+rand = "0.7.3"
```

Comment:
Why did `rand` get moved out of  `dev-dependencies`? Isn't `testutils.rs` behind a `#[cfg(test)]`?

Reply:
True, this was needed during some iteration on testutils and I forgot to clean this up.